### PR TITLE
Implement last of non-C'Thun WOG Warrior cards

### PR DESCRIPTION
**Pull request by astradamus**
Just Ancient Shieldbearer remains. From what I can gather, there's no C'Thun tracking in place just yet.

### CODE REVIEW

File: tests/test_wog.py [hunk 1]
```diff
@@ -166,6 +166,13 @@ def test_hallazeal_the_ascended():
 	assert hallazeal.dead
 
 
+def test_malkorok():
+	game = prepare_game()
+	malkorok = game.player1.give("OG_220")
+	malkorok.play()
+	assert game.player1.weapon
```
Comment (jleclanche) [inline tests/test_wog.py]: Check that the opponent doesn't have a weapon, as well.

File: tests/test_wog.py [hunk 2]
```diff
@@ -219,6 +219,14 @@ def test_silithid_swarmer():
 	assert not silithid.can_attack()
 
 
+def test_tentacles_for_arms():
+	game = prepare_game()
+	game.player1.discard_hand()
+	game.player1.give("OG_033").play()
+	game.player1.give(LIGHTS_JUSTICE).play()
+	assert game.player1.hand == ["OG_033"]
+
```
Comment (jleclanche) [inline tests/test_wog.py]: Can you add a separate `test_tentacles_for_arms_full_hand`?

Comment (astradamus) [inline tests/test_wog.py]: Sure. Are we making sure that it winds up in the graveyard instead of infinite looping?

File: tests/test_wog.py (hunk 1 above)
Comment (astradamus) [inline tests/test_wog.py]: Done.

**Pull request merged**